KomodoPlatform/atomicDEX-Desktop
# better min vol logic

closes https://github.com/KomodoPlatform/atomicDEX-Desktop/issues/2153

To test:
- Try to place order for/with KMD vs Smartchain at lowest amount possible.
- Check orders tab, one side will be 0.0001, and the other side larger than 0.0001 (as below)
![image](https://user-images.githubusercontent.com/35845239/206248571-fa551b34-007b-4f93-911c-b53dc29597e8.png)

- Look for a pair orderbook where maker has set custom minimum, and confirm can not start swap if setting volume lower
- Test other coin like DOGE or BTC, default minimum will be 0.1 or 0.00777 instead of 0.0001
